bears/python/PyImportSortBear.py

```diff
@@ -145,6 +206,9 @@ def run(self, filename, file,
             dependencies that occur in many projects.
         :param max_line_length:
             Maximum number of characters for a line.
+        :param treat_seperated_imports_independently:
+            Treat import statements seperated by one or more than one blank
```

Comment:
This is hard to read.
'Treat' is a meaningless word.
'one or more than one' is rarely used English. 'one or more' is simpler.